### DISCUSSION TITLE
separate blogs

### DISCUSSION THREAD

**Discussion by 14mC4** [squidfunk/mkdocs-material#4727] [Q&A] [answered]
Is there a way to make separate blogs? Basically on Jekyll I could have the categories split up and have one blog ties to one nav tab and the other blog tied to a separate blog tab.

basically
    - Home: index.md
    - HTB Writeups:
      - blog/index.md
     - THM Writeups:
      - blog/index.md
    Something like above. I've tried a few diff ways no avail.

**Comment by squidfunk** [comment, marked as answer]
Sure, so if [categories](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#adding-categories) are not enough to partition your content, you can use multiple instances of the blog plugin. You just need to use separate folders for your blog posts and set [`blog_dir`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#+blog.blog_dir) to their location.

**Reply by 14mC4** [reply]
I had tried that but cant seem to get it to server anymore after that

**Reply by 14mC4** [reply]
![image](https://user-images.githubusercontent.com/25674713/206880484-299d6246-e713-4ece-b168-40370501365e.png)

**Reply by 14mC4** [reply]
Tried relative and shorthand but, doesnt appear to by happy. High chance Im totally off though.

**Reply by 14mC4** [reply]
Well I thought. I did at least. Nevermind. See below Image for my current thought process. 
![image](https://user-images.githubusercontent.com/25674713/206880899-0471dcfd-2a49-4ca1-ace1-e4ebd2ffb4dd.png)

**Reply by squidfunk** [reply]
It's not clear to me if you figured it out entirely, but as mentioned in my last comment, you need separate instances of the blog plugin, which you can then configure separately:

``` yaml
plugins:
  - blog:
      blog_dir: htb
  - blog:
      blog_dir: thm
```

Then make sure those subdirectories exist, i.e. the directory layout must be:

```
.
├─ docs/
│  ├─ htb/
│  │  ├─ posts/
│  │  └─ index.md
│  └─ thm/
│     ├─ posts/
│     └─ index.md
└─ mkdocs.yml
```
